src/entities/entity_factory.cpp

```diff
@@ -4,12 +4,12 @@
 #include "brickengine/components/transform_component.hpp"
 #include "brickengine/components/colliders/rectangle_collider_component.hpp"
 #include "brickengine/components/physics_component.hpp"
-#include "brickengine/components/player_component.hpp"
+#include "components/player_component.hpp"
```

Comment:
Discuss with the group whether this is how we would like it, as it was in the engine before.